src/pages/about/ValuesBlock.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from 'styled-components';
+
+interface ValuesBlockProps {
+    content: {
+        title: string;
+        description: string;
+        image: {
+            picture: any;
+            alt: string;
+        };
+    }[];
+}
+
+const ValuesBlock = (props: ValuesBlockProps) => (
+    <ValuesBlockDiv>
+        {props.content.map((value, index) => (
+          <ValueDiv>
+          <ValueTitle>{value.title}</ValueTitle>
+          <ValueContent>
+            <Description>{value.description}</Description>
+            <Image src={value.image.picture} alt={value.image.alt}></Image>
+          </ValueContent>
+          </ValueDiv>
+        ))}
+    </ValuesBlockDiv>
+);
+
+const ValuesBlockDiv = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: 30px 220px 0px 220px;
+    & > div:nth-child(even) {
+        > p:first-child {
+            margin-left: auto;
+        }
+        > div {
+            
+        } 
+        > div > p {
+            order: 2;
+            text-align: right;
+        }
+        > div > img {
+            order: 1;
+            margin-left: auto;
+            margin-right: 50px;
+        }
+    }
+
+    @media only screen and (max-width: 1100px) {
+        > div > p:first-child {
+            margin: 0 auto;
+        }
+        > div > div { 
+            flex-direction: column;
+        }
+        > div > div > p {
+            order: 2;
+            text-align: center !important;
+            width: 100%;
+        }
+        > div > div > img {
+            order: 1;
+            margin: 30px auto !important;
+        }
+    } 
```

Comment:
to make this more readable, we can refer to the actual styled components instead of "div", "p", and "img"

for example:
`& > div:nth-child(even) { ... }` becomes `& > ${ValueDiv}:nth-child(even) { ... }`

and `> div > p {
            order: 2;
            text-align: right;
        }` becomes `> ${ValueContent} > ${Description} {
            order: 2;
            text-align: right;
        }`